src/main/java/client/Client.java

```diff
@@ -0,0 +1,29 @@
+package main.java.client;
+
+import java.io.*;
+import java.net.ServerSocket;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.net.ServerSocket'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=1&issues=AWQTU_q-bXXTsx7O43dX&open=AWQTU_q-bXXTsx7O43dX)

[ISSUE_KEY]: AWQTU_q-bXXTsx7O43dX